docs/SDL/OnStatusUpdate/assets/OnStatusUpdate_in_Proprietary_PTU_flow.txt

```diff
@@ -0,0 +1,32 @@
+title Policy Table Update (PROPRIETARY)
+
+participant Policy Server
+participant Mobile
+participant SDL
+participant HMI
+==<align:center>SDL is built with with **"-DEXTENDED_POLICY: PROPRIETARY"** flag==
+note over SDL: PT update is required
+
+SDL->HMI: SDL.OnStatusUpdate(UPDATE_NEEDED)
+SDL->SDL: Creates PT snapshot
+SDL->HMI: BC.PolicyUpdate(path_to_PTS)
+HMI-->SDL: SUCCESS: BC.PolicyUpdate
+SDL->HMI: SDL.OnStatusUpdate(UPDATING)
+SDL->SDL: Starts timeout taken from "timeout_after_x_seconds" of Local PT
+HMI->SDL: SDL.GetPolicyConfigurationData\n(policy_type=module_config, property=endpoints)
+note over SDL: Get endpoints from policies database
+SDL-->HMI: SDL.GetPolicyConfigurationData(value)

```

Comment:
Instead of just having `value` we could have
```suggestion
SDL-->HMI: SDL.GetPolicyConfigurationData\n(["{\"0x07\":{\"default\":[\"http://x.x.x.x:3000/api/1/policies\"]}, ...}"])
```
or something like that just to give the reader a better idea as to what type of data will be returned by the `SDL.GetPolicyConfigurationData` RPC